packages/element-library/src/video/constants.js

```diff
@@ -60,10 +60,10 @@ export const defaultAttributes = {
 };
 
 export const panels = [
-  PanelTypes.ELEMENT_ALIGNMENT,
+  PanelTypes.ElementAlignment,
```

Comment:
This is still constant though currently, not `enum`. 🤔

Reply:
No it isn't. https://github.com/GoogleForCreators/web-stories-wp/pull/12701/files#diff-8277fbfe1e012d5dffa9b90466147be94743b1a9e409e5e1245e9ab20cf19614R17

Reply:
Oh, true, my bad :)